contracts/dtoken/src/lib.rs

```diff
@@ -68,6 +74,41 @@ impl Dtoken {
     pub fn get_underlying_balance() -> u128 {
         BigDecimal::from_str("1.2").unwrap().to_u128().unwrap()
     }
+
+    pub fn internal_unwrap_balance_of(&self, account_id: &AccountId) -> Balance {
+        self.token.internal_unwrap_balance_of(account_id)
+    }
+
+    pub fn internal_deposit(&mut self, account_id: &AccountId, amount: Balance) {
+        self.token.internal_deposit(account_id, amount);
+    }
+
+    pub fn internal_withdraw(&mut self, account_id: &AccountId, amount: Balance) {
+        self.token.internal_withdraw(account_id, amount);
+    }
+
+    pub fn internal_transfer(
```

Comment:
@TymofiiSavinov interface should follow NEP21 standard https://nomicon.io/Standards/FungibleToken/Core.html

internal_transfer -> ft_transfer
sender_id should be implicit, this is id of the account that called ft_transfer could be retrived from the env